src/test/kotlin/org/jetbrains/research/transformations/SetupParser.kt

```diff
@@ -0,0 +1,50 @@
+package org.jetbrains.research.transformations
+
+import java.io.File
+import java.util.logging.Logger
+
+object SetupParser {
+    val LOG = Logger.getLogger(this.javaClass.name)
+    private const val TARGET_PATH = "/tmp/pythonparser"
+
+    fun putParserToTmp() {
```

Comment:
It's better to call it "putParserToTargetPath" because once your TARGET_PATH changes you'll need to change the name of this function too and that's not great (also maybe you should create an argument "targetPath" and set the default value as TARGET_PATH to improve reusability)

Reply:
great idea, thank you